index.js

```diff
@@ -37,8 +38,36 @@ module.exports = (filePath, options) => {
 	if (optionsObj.verbose) {
 		console.log(chalk.gray(cmd));
 	}
-
+	var diff = [];
 	const resultBuffer = execSync(cmd);
+	const other_cmd = "grep -inRw -E 'TODO' . --exclude-dir={node_modules,test}"
```

Comment:
Can we leverage the existing grep command above so we're not searching Todo's twice?